### DISCUSSION TITLE
Review all?

### DISCUSSION THREAD

**Discussion by thorenslove** [asreview/asreview#1388] [Q&A]
Do I have to review all the dataset for project? I understand a couple of review in relevant or not could train AI model that allows outcome relevant only.

**Comment by rohitgarud** [comment]
Hello @thorenslove. The primary purpose of ASReview is to reduce the number of records to be reviewed at the screening stage so that we don't have to read all the abstracts at the screening stage. According to the [published research](https://doi.org/10.1038/s42256-020-00287-7) by ASReview developers, only 8-33% of records need to be reviewed to discover 95% of the relevant records. However, it cannot guarantee that after giving it a couple of relevant records, it will only outcome relevant records as the ASReview is based on active learning and it learns what it means to be relevant records as you label the records as relevant and irrelevant. This process requires labeling multiple records as there are nuanced differences between the relevant and irrelevant records.
